### DISCUSSION TITLE
Conversation history lost.

### DISCUSSION THREAD

**Discussion by zwang98** [lencx/ChatGPT#581] [Q&A] [answered]
I was playing around the chat and suddenly lost all of the past conversations.
Also, it is not able to save any conversation as a history now:

![image](https://user-images.githubusercontent.com/56372651/223607472-4024ebff-14d4-491e-90eb-3e597f2a943d.png)

Chat.download is a null list:
![im
[ChatGPT.log](https://github.com/lencx/ChatGPT/files/10916303/ChatGPT.log)
age](https://user-images.githubusercontent.com/56372651/223607574-82938e9d-0182-45a6-99b2-5efb3d452b7d.png)

**Comment by zwang98** [comment]
Attach log here, some errors insides:
[ChatGPT.log](https://github.com/lencx/ChatGPT/files/10916323/ChatGPT.log)

**Comment by Bauklotze1** [comment]
I currently have this problem too  Conversation history lost

**Comment by pete-liao** [comment]
I am also experiencing a loss of chat history, and the language settings display no data.

**Comment by kts273** [comment]
My chat history disappeared and can't record the new one as well. Additionally,  it is empty in the "Set Speech Language."

**Comment by lencx** [comment, marked as answer]
This issue is from OpenAI and has been reported in the `OpenAI Discord`, but no one has replied yet.
<img width="859" alt="bug" src="https://user-images.githubusercontent.com/16164244/223629534-8c355141-c322-42d3-85fb-ead6a08743e6.png">

**Reply by lencx** [reply]
OpenAI has fixed this issue, log out of your account, then log back in and you'll see the chat history.

**Reply by zwang98** [reply]
thx for noticing and reply

**Reply by andesmountain** [reply]
did not work! i tried everything, log out and in , refresh page ,clear web cache and cookie, change web browser ....

**Comment by mpardi64** [comment]
Your chagpt history has not gone, retrieve it from your browser history
